tests/test_export_testcase.py

```diff
@@ -0,0 +1,54 @@
+import os
+import unittest
+
+import chainer
+import chainer.functions as F
+import chainer.links as L
+import numpy as np
+
+import onnx_chainer
+
+
+class TestExportTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.model = chainer.Sequential(
+            L.Convolution2D(None, 16, 5, 1, 2),
+            F.relu,
+            L.Convolution2D(16, 8, 5, 1, 2),
+            F.relu,
+            L.Convolution2D(8, 5, 5, 1, 2),
+            F.relu,
+            L.Linear(None, 100),
+            F.relu,
+            L.Linear(100, 10)
+        )
+        x = np.zeros((1, 3, 28, 28), dtype=np.float32)
+        self.x = chainer.as_variable(x)
+
+    def test_export_testcase(self):
+        # Just check the existence of pb files
+        path = 'out/test_export_testcase'
```

Comment:
This code works on Windows, but it's better to use `os.path.join` and `"out"` is got from `onnx_chainer.testing.test_onnxruntime.TEST_OUT_DIR`. However related to this directory will be fixed in #117 and next PR , so it's fine like this.